### PR TITLE
Update autograph.ipynb

### PR DESCRIPTION
**Pull request by mdanatg**
Final updates to the tutorial, to ensure smooth functioning under the current nightly pip. Most of these changes should be retired within a week or two, as the nightly updates.

### CODE REVIEW

**Comment by mdanatg** [comment]
With these changes, the notebook should execute as loaded by just hitting Connect and running all cells.

**Comment by mdanatg** [comment]
@brilee FYI

**Comment by MarkDaoust** [comment]
LGTM

File: site/en/2/guide/autograph.ipynb [hunk 1]
```diff
@@ -646,18 +672,20 @@
...
+        "@tf.function\n",
         "def train(model, optimizer, train_ds):\n",
         "  step = 0\n",
         "  for x, y in train_ds:\n",
         "    step += 1\n",
         "    loss, accuracy = train_one_step(model, optimizer, x, y)\n",
         "    if step % 10 == 0:\n",
-        "      print(\n",
-        "          'Step', step, ': loss', loss.numpy(), '; accuracy', accuracy.numpy())\n",
+        "      print('Step', step, ': loss', loss, '; accuracy', accuracy)\n",
```
[the remark this answers is not in the document]
Comment (mdanatg) [reply site/en/2/guide/autograph.ipynb]: It's meant to use semicolon: "Step n : loss X ; accuracy Y"

**Pull request merged**